text/0000-clearer-changelogs.md

```diff
@@ -0,0 +1,142 @@
+
+This could be opt-in, PRs that contain a _Description_ (or _Changelog_, or similar) heading could have that description automatically added to the appropriate changelog. Gatsby contributors or maintainers should add descriptions to PRs where necessary.
+
+Descriptions could vary in length depending on the PR. A new feature might have a longer description (with a link out to related docs), where a typo fix might not need any description.
+
+### Improve version bump message
+
+Often a changelog entry will be the line "Note: Version bump only for package <packagename>", currently this will happen if the PR title prefix is not `feat`, `fix` or `perf`. 
+
+Modify the changelog generation preset we're using to provide information in place of this message.
+
+### Streamline category display
+
+Most releases have less than five items (often, just one item) in them, making the category headings feel unnecessary. A couple of changes to improve readability:
+
+- replace category headings (bug fix, feature) with a sorted, prefixed list of changes
+- add distinct, per-category emojis for easy scanning :eyes: (todo: define an emoji for each category)
+
+### Remove brackets around related PR link
+
+A small change, but helps to separate it from the PR author link.
+
+### Remove link to the specific commit
+
+This doesn't provide any additional information over the PR link. Remove it.
+
+### Credit pull request author(s)
+
+Contributors are nice, let's credit them for their work.
+
+## Other changes
+
+### Update monorepo root changelog
+
+There's a [(mostly unused) root changelog](https://github.com/gatsbyjs/gatsby/blob/master/CHANGELOG.md) which should be updated to point to the individual changelogs.
+
+In the future it might be interesting to compile the individual package changelogs into the root changelog, to provide a nice overview of all changes in the Gatsby repo. This is probably a discussion for a separate issue.
+
+# Drawbacks
+
+All of the information presented in the changelog is already available by hunting around GitHub for it. Instead of spending time implementing this RFC, we could work on features and bug fixes instead.
+
+The optional (but encouraged) descriptions require the most work to implement, and could be dropped without affecting the rest of this RFC.
+
+# Alternatives
+
+There are many ways to format a changelog, this is just one of them.
+
+# Adoption strategy
+
+The optional (but encouraged) descriptions would need some extra tooling support. In addition, maintainers (and contributors) would need to be informed how to write a good description, and how the description would be used. 
+
+# How we teach this
+
+Mostly N/A as this is related to updating tooling. For the parts that need to be taught, we could use a combination of gatsbot, an updated PR template, and documentation on gatsbyjs.org.
+
+# Unresolved questions
+
+Are the benefits of additional descriptions worth the extra work?
```

Comment:
I feel like they're one of the biggest benefits of doing this. But we can do everything else first, and then follow up with the additional descriptions at a later date.

Reply:
I would agree with this. In my mind we should aspire to have a changelog that people take screenshots of and share — Yelp is really good at this, for example — because it A) encourages people to read them, and B) helps ensure that the content is actually useful.

We don't necessarily need to hit that mark, but if we start with that goal in mind and scale back from there, I think we'll end up in a good place.

Reply:
Just to clarify - is the question about extra work do adjust tools or extra work for every PR to compose concise/precise additional description?